.gitignore

```diff
@@ -26,9 +26,7 @@ pids
 *.sln
 *.sw?
 .DS_Store
-.idea/gradle.xml
-.idea/misc.xml
-.idea/vcs.xml
+.idea

```

Comment:
This file is applied not just to the generated projects, but to this repo itself as well. We already have some files in .idea folder in this repo. This would mean:
* changes to those files in this repo are ignored
* generated .idea folder (which helps with Kotlin syntax as far as I know) will only be present on the machine the zip is extracted on, but not for anyone else working on that project (who'd be pulling from a repo).

If we want 2, then we would want to keep these files editable in this repo. In which case we'd want slightly different .gitignore files for this repo itself and for the generated projects.

If we don't want 2, then we should double-check that want this .idea folder in this repo. if we get rid of it, then adding .idea to .gitignore would be fine.

Reply:
Ah, I see the problem. I think we need 2 but need to think about how resulting project will ignore .idea as it's causing pain / manual correction required in each app built using genx and Create.